src/main/java/eu/europeana/metis/sandbox/controller/PatternAnalysisController.java

```diff
@@ -23,14 +23,18 @@
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
 import java.time.LocalDateTime;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
 import java.util.Map.Entry;
+import java.util.Optional;
+import java.util.Set;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.locks.Lock;
 import java.util.concurrent.locks.ReentrantLock;
 import java.util.stream.Collectors;
-
-import org.jetbrains.annotations.NotNull;
+import javax.validation.constraints.NotNull;
```

Comment:
I think this is the right import ```org.jetbrains.annotations.NotNull;```I don't know why it is changed here.

Reply:
I believe not. Shouldn't it be a javax.validation.not NULL?!
So our code is dependent Intellij jetbrains, cause it's not a maven dependencyd

Reply:
yes. I confirmed  the documentation. https://docs.oracle.com/javaee/7/api/javax/validation/constraints/NotNull.html